js/packages/cli/src/commands/mint-nft.ts

```diff
@@ -92,6 +110,45 @@ export const createMetadata = async (
   });
 };
 
+export const createMetadataAccount = async ({
+  connection,
+  data,
+  mintKey,
+  walletKeypair,
+}: {
+  connection: Connection;
+  data: DataV2;
+  mintKey: PublicKey;
+  walletKeypair: Keypair;
+}): Promise<PublicKey | void> => {
+  // Retrieve metadata
+  const metadataAccount = await getMetadata(mintKey);
```

Comment:
Do you need to add any sort of error handling here? try catch? Nothing guarantees metadataAccount is valid.